<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::TestDeviceFinished</class>
 <widget class="QWidget" name="ns::TestDeviceFinished">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetTestFinished {
	background: #ffffff;
	border-bottom : 1px solid #eff4f6;
}

#labelTestIcon {
	background-image: url(:/images/ui/test_normal.png);
}

#labelTestResultTitle {
	font-size: 15px;
	font-weight: bold;
}

/*#widgetResult QLabel {
	border: 1px solid #eeeeee;
}*/

#labelSpeakerResultKey, #labelMicResultKey, #labelCameraResultKey, #labelNetworkResultKey, #labelNetQualityResultKey {
	background: #FBFBFB;
	border-top: 1px solid #eeeeee;
	border-right: 1px solid #eeeeee;
	border-left: 1px solid #eeeeee;
}

#labelNetQualityResultKey {
	border-bottom: 1px solid #eeeeee;
}

#labelSpeakerResultValue, #labelMicResultValue, #labelCameraResultValue, #labelNetworkResultValue, #labelNetQualityResultValue {
	border-top: 1px solid #eeeeee;
	border-right: 1px solid #eeeeee;
}

#labelNetQualityResultValue {
	border-bottom: 1px solid #eeeeee;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTestFinished" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTestIcon">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTestResultTitle">
          <property name="minimumSize">
           <size>
            <width>390</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>390</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>The test is completed, your computer can provide translation services to normal</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>130</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widgetResult" native="true">
          <property name="minimumSize">
           <size>
            <width>496</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>496</width>
            <height>140</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelSpeakerResultKey">
               <property name="minimumSize">
                <size>
                 <width>145</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>145</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Speaker</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelSpeakerResultValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelMicResultKey">
               <property name="minimumSize">
                <size>
                 <width>145</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>145</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Microphone</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelMicResultValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelCameraResultKey">
               <property name="minimumSize">
                <size>
                 <width>145</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>145</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Camera</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelCameraResultValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelNetworkResultKey">
               <property name="minimumSize">
                <size>
                 <width>145</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>145</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Network speed</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelNetworkResultValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelNetQualityResultKey">
               <property name="minimumSize">
                <size>
                 <width>145</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>145</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Network quality</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelNetQualityResultValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
